packages/common/mui/src/lib/data-table-cells/multi-value-cell/multi-value-cell.tsx

```diff
@@ -12,38 +13,51 @@ export const MultiValueCell = ({ cell }): JSX.Element => {
   const FirstValue = column?.expandableTable ? LargeTableCell : TableCell;
   const SecondValue = column?.expandableTable ? TableCell : SmallTableCell;
 
+  const amountPaidPending =
```

Comment:
Can we rename this to be less specific? Shouldn't this just be like isPending or something? It also applies to other cells as well?